README.md

```diff
@@ -75,6 +75,9 @@ DSN for [Sentry](https://sentry.io/welcome/) to automatically track runtime erro
 
 Tracking ID of a Google Analytics Property to monitor usage of the Hub.
 
+    hub_offering: "lean"
+Offering setting defining the menu layout. Default is "lean" used to only show a subset of the features in opposed to "full".
```

Comment:
missing blank line.

Reply:
please check the other variables and apply consistent style in formatting and wording.